cli/command/stack/swarm/deploy_composefile_test.go

```diff
@@ -35,6 +35,7 @@ func TestValidateExternalNetworks(t *testing.T) {
 			expectedMsg:  "Unexpected",
 		},
 		{
+			// FIXME(vdemeester) that doesn't work under windows, the check needs to be smarter
```

Comment:
This tests fails under Windows because of : https://github.com/docker/cli/commit/7f53c99dfe5241a985dd5ce4bed6c116a2908b41#r27815425

Reply:
I think the info endpoint could be queried instead of making the assumption that daemon os/platform == client os/platform

Reply:
definitely, but probably need to be "skipped" for now, and follow-up on a fix PR for that problem only 👼